Packs/Censys/.pack-ignore

```diff
@@ -1,2 +1,5 @@
 [file:Censys.yml]
-ignore=IN145
\ No newline at end of file
+ignore=IN145
+
+[file:README.md]
+ignore=RM102
```

Comment:
why is this needed?

Reply:
I'm giving one command example for each. And I have outputs that are not in the readme. 
For example, running view with host is will give the outputs for the host but in the yml we have also the certs outputs which is not in the example